app/main.py

```diff
@@ -1,3 +1,117 @@
+import json
+
```

Comment:
the blank line does not need, all imports from the same category -built in libraries